docs/root/configuration/http/http_filters/aws_request_signing_filter.rst

```diff
@@ -0,0 +1,42 @@
+
+.. _config_http_filters_aws_request_signing:
+
+AWS Request Signing
+===================
+
+* :ref:`v2 API reference <envoy_api_msg_config.filter.http.aws_request_signing.v2alpha.AwsRequestSigning>`
+* This filter should be configured with the name *envoy.filters.http.aws_request_signing*.
+
+.. attention::
+
+  The Aws request signing filter is experimental and is currently under active development.
+
+The HTTP Aws request signing filter is used to access authenticated AWS services. It uses the
```

Comment:
nit: s/Aws/AWS, similar elsewhere.

Reply:
hmm but just in docs right? In code we want `Aws`? cc: @derekargueta